.repo-tools/Docs_Automation/guids.json

```diff
@@ -923,5 +923,15 @@
         "guid": "dc8c1c92-1c2a-4edf-abbd-6c35dfab256a",
         "ver": "2021.4.0",
         "name": "VisualComputing tutorial"
+    },
+    "748eb906-309f-469b-95c9-5258006c7587": {
+        "guid": "748eb906-309f-469b-95c9-5258006c7587",
+        "ver": "2021.4.0",
+        "name": "CRR"

```

Comment:
This design is not new in 2021.4.  I'm not sure when it was first added, but this is not accurate.

Reply:
fixed - 2021.1.Gold